plantshop.py

```diff
@@ -10,6 +10,16 @@ def grow(self, no_inches):
     def wither(self, how_many_leaves_fall):
         self.number_of_leaves -= how_many_leaves_fall
 
+    def propagate(self, number_of_weeks):
+        if (number_of_weeks > 3):
+            print(number_of_weeks)
```

Comment:
Maybe we can print a statement instead of just a number here. Let's print something like "Hurry up and propagate, the plant is getting old"

Reply:
sure, no problem